roles/vdm/templates/transformers/launcher-nfs-mount.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: builtin
+kind: PatchTransformer
+metadata:
+  name: sas-launcher-nfs-mount
```

Comment:
Wondering about the name here. The older files all have the file name that matches the metadata name. This one is different. Did the dev team change the naming or are we?

Reply:
I didn't change the naming. I'm using the metadata name specified in the transformer from the dev team at sas-bases/examples/sas-launcher/configure/launcher-nfs-mount.yaml

```
###################################################################################
# Kustomize patch configuration to change the add the launcher.sas.com/nfs-server
# PodTemplate annotation. This allows you to configure Viya with the location of
# the NFS server hosting a user's home directories at deployment time. The path
# is determined by the identities service.
# See README.md for more information.
###############################################################################
---
apiVersion: builtin
kind: PatchTransformer
metadata:
  name: sas-launcher-nfs-mount
patch: |-
  - op: add
    path: /metadata/annotations/launcher.sas.com~1nfs-server
    value: {{ NFS-SERVER-LOCATION }}
target:
  kind: PodTemplate
  labelSelector: sas.com/template-intent=sas-launcher
```